src/FSharpVSPowerTools.Logic/VSUtils.fs

```diff
@@ -118,6 +102,59 @@ type ITextSnapshot with
     /// Get the text at line `num`
     member inline x.LineText num =  x.GetLineFromLineNumber(num).GetText()
 
+
+type SnapshotSpan with
+
+    member inline x.StartLine  = x.Snapshot.GetLineFromPosition (x.Start.Position)
+    member inline x.StartLineNum  = x.Snapshot.GetLineNumberFromPosition x.Start.Position
+    member inline x.StartColumn  = 
+        x.Start.Position - x.StartLine.Start.Position 
+    
+    member inline x.EndLine = x.Snapshot.GetLineFromPosition (x.End.Position)
+    member inline x.EndLineNum  = x.Snapshot.GetLineNumberFromPosition x.End.Position
+    member inline x.EndColumn = 
+        x.End.Position - x.EndLine.Start.Position          
+
+    member x.ModStart (num) =
+        SnapshotSpan(SnapshotPoint (x.Snapshot, x.Start.Position + num), x.End)
+
+    member x.ModEnd (num) =
+        SnapshotSpan(x.Start, (SnapshotPoint (x.Snapshot,x.End.Position + num)))
+
+    member x.ModBoth m1 m2 =
+        SnapshotSpan(SnapshotPoint (x.Snapshot, x.Start.Position + m1)
+                    ,SnapshotPoint (x.Snapshot, x.End.Position + m2))
```

Comment:
Please add XML Doc comments for these `ModXXX` members. It's unclear what "Mod" means.